webserver/HuConJsonRpc.py

```diff
@@ -199,13 +208,21 @@ def _get_file_list(self, rpc_request):
         """
         try:
             code_folder = os.path.join(self._CODE_ROOT, rpc_request['params'].strip('/\\'))
+            example_folder = os.path.join(self._EXAMPLE_ROOT, rpc_request['params'].strip('/\\'))
 
+            files_usercode = []
             rpc_response = self._get_rpc_response(rpc_request['id'])
-            rpc_response['result'] = os.listdir(code_folder)
+            if os.path.exists(code_folder):
+                files_usercode = os.listdir(code_folder)
+                files_usercode.sort()
```

Comment:
Do you need to sort here?

Reply:
As the file-system may give us the files in an unordered fashion sorting seems necessary.